### PR TITLE
Tweak actions menu

### PR DESCRIPTION
**Pull request by silverwind**
Ressurect lost changes from https://github.com/go-gitea/gitea/pull/24451.

- Always show icons for each entry in the menu
- Make all checkboxes toggle only their feature, e.g. "seconds" and "timestamps" can now be toggled on together.
- Reorder the items

Screenshot 2023-08-01 at 19 19 27

### CODE REVIEW

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -403,18 +403,9 @@ const sfc = {
...
-      if (this.timeVisible[`log-time-${type}`] && this.timeVisible[`log-time-${other}`]) {
-        this.timeVisible[`log-time-${other}`] = false;
-        toToggleTypes.push(other);
-      }
-      for (const toToggle of toToggleTypes) {
-        for (const el of this.$refs.steps.querySelectorAll(`.log-time-${toToggle}`)) {
-          toggleElem(el, this.timeVisible[`log-time-${toToggle}`]);
-        }
+      for (const el of this.$refs.steps.querySelectorAll(`.log-time-${type}`)) {
+        toggleElem(el, this.timeVisible[`log-time-${type}`]);
```
Comment (wxiaoguang) [inline web_src/js/components/RepoActionView.vue:407]: See the comment above: `    // show at most one of log seconds and timestamp (can be both invisible)`

Comment (silverwind) [reply web_src/js/components/RepoActionView.vue:407]: Comment removed, it is obsolete now that those two can toggle individually.

**Review by denyskon — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]
IIRC the old logic ("show at most one of log seconds and timestamp (can be both invisible)") was from @HesterG 

While I can accept either.

**Comment by HesterG** [comment]
I am good with both as long as other maintainers are OK with this.

**Review by delvh — APPROVED** [review]

**Pull request merged**